timer.js

```diff
@@ -4,6 +4,8 @@ let secondsLeft;
 let totalSeconds;
```

Comment:
I really liked that you have part of the logic in a separate file. This is something you will see in React when modularizing the components of your app, and it is very useful for keeping the code organized and easier to read. Bravo! 🙌